decentraland/unity-explorer
# fix: bind connection status panel visibility to debug command

## What does this PR change?



Fix #2439 

Enable the connections status button only if the /debug command is used.
The visibility is bound to the debug menu.

## How to test the changes?



1. Launch the explorer
2. Check that the button is not displayed
3. Send the /debug command
4. Check that the button is now displayed
5. Send another /debug command
6. Check that the button is now hidden

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
